src/main/resources/migrations/V1__Initial.sql

```diff
@@ -1,7 +1,7 @@
-CREATE TABLE IF NOT EXISTS spark.company
+CREATE TABLE IF NOT EXISTS company
```

Comment:
you shouldn't update previous migrations. It can cause to some problems if this script has been already migrated.
It works only on clear database

Reply:
I use a new DB in a docker container. I used another DB on port 5432 before.

Reply:
I use a new DB in a docker container. I used another DB on port 5432 before.

Reply:
Ok. But in the next PR's you have to avoid it